### PR TITLE
Refactor sfi messages to allow customization

### PR DESCRIPTION
**Pull request by patari**
#### Summary

Messages sent by the app includes words like "Espoo" and "eVaka" which needs to customized for other installations.

### CODE REVIEW

**Comment by Gekkio** [comment]
OWASP errors have been fixed in our master -> this needs rebase

**Comment by patari** [comment]
Done, thanks!

File: service/src/main/kotlin/fi/espoo/evaka/shared/message/MessageProvider.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.shared.message
+
+interface IMessageProvider {
+    fun get(type: MessageType, lang: MessageLanguage): String
```
Comment (Gekkio) [inline service/src/main/kotlin/fi/espoo/evaka/shared/message/MessageProvider.kt:8]: Is there some advantage to using one function + enum with N variants + when checks, vs just using N functions? The enum solution seems more complicated to me.

In other words:

```kotlin
enum class MessageType {
  A,
  B
}

fun get(type: MessageType, lang: MessageLanguage) {
  return when(type) {
    MessageType.A -> // ...
    MessageType.B -> // ...    
  }
}
```

vs

```kotlin
fun getA(lang: MessageLanguage) {
  // ...
}
fun getB(lang: MessageLanguage) {
  // ...
}
```

Comment (patari) [reply service/src/main/kotlin/fi/espoo/evaka/shared/message/MessageProvider.kt:8]: Yeah, I think you're right. I'll change this

Comment (patari) [reply service/src/main/kotlin/fi/espoo/evaka/shared/message/MessageProvider.kt]: Done

**Review by Gekkio — APPROVED** [review]

**Pull request merged**